phpreport-report

```diff
@@ -330,11 +331,20 @@ class DetailedReport(Report):
 
         all_stories = [get_story(task) + task.text for task in tasks_for_day]
 
-        # Many times people add a lot of duplicate descriptions. Just output one of each.
-        all_stories = set(all_stories)
+        # Many times people add a lot of duplicate descriptions. Just output
+        # one of each, keeping the stories ordering
```

Comment:
Nit: "keeping the stories ordering" should probably read "keeping the stories in the same order."

Reply:
BTW, this seems like a separate change. Do you mind opening up a PR just for this? I'd like to commit it apart from preserving the newlines.